README.md

```diff
@@ -326,6 +326,26 @@ Obtain SMTP credentials from a provider for transactional emails. Set the SMTP_U
 
 <hr>
 
+<img src="https://upload.wikimedia.org/wikipedia/commons/4/44/Microsoft_logo.svg" height="50">
```

Comment:
This section seems to be too long compared to sections for other providers; you can probably trim it a bit.